.woodpecker.yml

```diff
@@ -1,56 +1,35 @@
 variables:
-  - &rust_image "rust:1.78"
+  - &rust_image "rust:1.78-slim-bookworm"
 
 steps:
-  prepare_repo:
-    image: alpine:3
-    commands:
-      - apk add git
-      - git submodule init
-      - git submodule update
-    when:
-      - event: [pull_request, tag]
-
-  prettier_check:
-    image: tmknom/prettier:3.2.5
-    commands:
-      - prettier -c .
-    when:
-      - event: pull_request
```

Comment:
Removed because none of the file types in this project are affected by prettier.